LibraBFT/Impl/Consensus/RoundManager.agda

```diff
@@ -390,3 +390,42 @@ newTcAggregatedM now tc =
   BlockStore.insertTimeoutCertificateM tc >>= λ where
     (Left e)     → logErr e
     (Right unit) → processCertificatesM now
+
+------------------------------------------------------------------------------
+
+-- TODO-1 PROVE IT TERMINATES
+{-# TERMINATING #-}
+xxx : BlockRetrievalRequest → (Author × Epoch × Round) → BlockStore → List Block → HashValue
+    → BlockRetrievalResponse
+
+-- external entry point
+processBlockRetrievalRequestM : Instant → BlockRetrievalRequest → LBFT Unit
+processBlockRetrievalRequestM _now request = do
+  -- logEE lSI (here []) $ do
+  use (lRoundManager ∙ rmObmMe) >>= λ where
+    nothing → logErr fakeErr -- should not happen
```

Comment:
Did you update your Haskell repo.
In Haskell,`rmObmMe` now returns `Maybe Author`

In Haskell I also added `rmObmMeUnsafe`.
At some point it would be good to refactor (non-test) places that use `rmObmMeUnsafe` to use `rmObmMe` (the safe one).

Reply:
Yes, I did update, and checked again just now.  Unless I am confused, the Haskell code still uses `rmObmMeUnsafe` here, which differs from the Agda.

Reply:
Oops - got my changes, commits, pushes confused.
I just pushed to Haskell - it  use the safe version now.